src/Persistence/Array_.php

```diff
@@ -293,11 +282,8 @@ public function initAction(Model $m, $fields = null)
 
     /**
      * Will set limit defined inside $m onto data.
-     *
-     * @param Model         $m
-     * @param ArrayIterator $action
      */
-    protected function setLimitOrder($m, $action)
+    protected function setLimitOrder(Model $m, \atk4\data\Action\Iterator &$action)
```

Comment:
Why reference markup - $action is object so it should be already passed as reference.